decentraland/screens/MyAssetsScreen/MyAssetsScreen.test.js

```diff
@@ -5,12 +5,22 @@ import { parcel, parcelUserAction } from "@helpers/testHelpers";
 
 describe("ListLandForSaleScreen", () => {
   describe("renders the component", () => {
+    const removeFromMyAssetsList = () => {};
+    const appendToMyAssetsList = () => {};
+    const addUserAction = () => {};
```

Comment:
This part wasn't automatic, and it wasn't just alphabetization like a lot of the other changes